cfgov/v1/signals.py

```diff
@@ -29,17 +29,13 @@ def new_phi(user):
 
 def user_save_callback(sender, **kwargs):
     user = kwargs['instance']    
-
-    try:
-        current_password_data = user.passwordhistoryitem_set.latest()
-    
-        if user.password != current_password_data.encrypted_password:
+    if kwargs['created']:
+        # If user was just created, expire the password and unlock it
+        new_phi(user, expiration_days=0, locked_days=0)
+    else:
+        if user.password != user.passwordhistoryitem_set.latest().encrypted_password:
             new_phi(user)
 
```

Comment:
By the way, getting rid of the `try` block here because it seems like this was only needed to check the condition where the user was a newly created one, but if that's not the case do chime in!